docs/tutorials/training_to_inference.md

```diff
@@ -0,0 +1,201 @@
+# DALI - from training to inference
+
+This tutorial presents the simple way of going from training to inference. We are focusing on a particular scenario: 
+how to deploy a model, when it has been trained using DALI as a preprocessing tool. An example that constitutes this 
+tutorial is the [EfficientNet](https://github.com/NVIDIA/DALI/tree/main/docs/examples/use_cases/pytorch/efficientnet) 
+from DALI repository. This tutorial is split in two parts - [Theory](#Theory) and [Practice](#Practice). Lastly, while we cover only the 
+deployment of DALI preprocessing graph, we are using handful of other NVIDIA tools useful for setting up the inference. 
+Documentation and links to those tools will be provided, but the specifics of these tools won't be explained - should 
+you like to know more details about them, please refer to the documentation.
+
+## Assumptions
```

Comment:
I feel like this section could be a first subsection of the Practice part. The theory is more general and those assumptions refer to a specific use case

Reply:
Good point. My reasoning was that these assumptions also refer to the `theory` part. Especially: `@pipeline_def` requirement on the pipeline definition, using Triton for the inference orchestration and putting preprocessing on the server side. Do you feel that the Theory part without emphasising these would be understandable?

Reply:
Hmm. For me the theory part is understandable, but it works both way, so maybe keep according to your intuition.